### PR TITLE
Landing improvements

### PR DESCRIPTION
**Pull request by kellan-simiotics**

### CODE REVIEW

File: frontend/src/components/ExampleCode.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+            {"}"}
+          </code>
+        </pre>
+      </>
+    );
+  };
+
+  return (
+    <Flex
+      // h={["auto", "auto", "320px"]}
```
Comment (peersky) [inline frontend/src/components/ExampleCode.js:104]: remove comments

File: frontend/src/components/ExampleCode.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+      flexDirection="column"
+      textColor="white"
+      bgColor="#686464"
+      p="20px"
+      rounded="lg"
+    >
+      <Heading as="h3" fontSize="lg" pb="20px">
+        We make sure our code is easy to use. Here’s an example:
+      </Heading>
+      {/* <Flex dangerouslySetInnerHTML={formattedCode}></Flex> */}
```
Comment (peersky) [inline frontend/src/components/ExampleCode.js:114]: remove comments

File: frontend/src/components/ExampleCode.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+      <Text fontSize="md">
+        {" "}
+        &nbsp; &nbsp;
+        {
+          "let activeOpening = await checkUsersActiveLootboxOpeningStatus( userAddress );"
+        }
+      </Text>
+      <Text fontSize="md">&nbsp; &nbsp;{"if ( activeOpening != null ) {"}</Text>
+      <Text fontSize="md">
+        &nbsp; &nbsp; &nbsp; &nbsp;
```
Comment (peersky) [inline frontend/src/components/ExampleCode.js:136]: remove comments

File: frontend/src/core/providers/OverlayProvider/index.js [hunk 4]
```diff
@@ -56,6 +56,8 @@ const NewSubscription = React.lazy(() =>
 
 const UploadABI = React.lazy(() => import("../../../components/UploadABI"));
 
+// const ExampleCode = React.lazy(() => import("../../../components/ExampleCode"));
```
Comment (peersky) [inline frontend/src/core/providers/OverlayProvider/index.js:59]: remove comments

File: frontend/src/core/providers/OverlayProvider/index.js [hunk 5]
```diff
@@ -322,6 +324,9 @@ const OverlayProvider = ({ children }) => {
               {modal.type === MODAL_TYPES.MOBILE_INPUT_FIELD && (
                 <UpdateSubscriptionLabelInput {...modal.props} />
               )}
+              {/* {modal.type === MODAL_TYPES.EXAMPLE_CODE && (
```
Comment (peersky) [inline frontend/src/core/providers/OverlayProvider/index.js:327]: remove comments

File: frontend/src/core/providers/OverlayProvider/constants.js [hunk 6]
```diff
@@ -10,6 +10,7 @@ export const MODAL_TYPES = {
   UPLOAD_ABI: 8,
   NEW_DASHBOARD_FLOW: 9,
   MOBILE_INPUT_FIELD: 10,
+  EXAMPLE_CODE: 11,
```
Comment (peersky) [inline frontend/src/core/providers/OverlayProvider/constants.js:13]: This is unused, right ? Remove

**Comment by peersky** [comment]
```
react_devtools_backend.js:4026 Warning: Received NaN for the `height` attribute. If this is expected, cast the value to a string.
    at svg
    at _RoundedRectSVG
```

**Comment by peersky** [comment]
react-dom.development.js?61bb:630 Error: <path> attribute d: Expected number, "… 16 16 C 16 16, NaN NaN, NaN NaN".
setValueForProperty @ react-dom.development.js?61bb:630
updateDOMProperties @ react-dom.development.js?61bb:8949
updateProperties @ react-dom.development.js?61bb:9368
commitUpdate @ react-dom.development.js?61bb:10252
commitWork @ react-dom.development.js?61bb:21401
commitMutationEffects @ react-dom.development.js?61bb:23401
callCallback @ react-dom.development.js?61bb:3945
invokeGuardedCallbackDev @ react-dom.development.js?61bb:3994
invokeGuardedCallback @ react-dom.development.js?61bb:4056
commitRootImpl @ react-dom.development.js?61bb:23121
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
commitRoot @ react-dom.development.js?61bb:22990
performSyncWorkOnRoot @ react-dom.development.js?61bb:22329
eval @ react-dom.development.js?61bb:11327
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
flushSyncCallbackQueueImpl @ react-dom.development.js?61bb:11322
flushSyncCallbackQueue @ react-dom.development.js?61bb:11309
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21893
dispatchAction @ react-dom.development.js?61bb:16139
checkForUpdates @ use-subscription.development.js?34f6:85
eval @ loadable.js?3b89:183
_update @ loadable.js?3b89:183
eval @ loadable.js?3b89:164
Promise.then (async)
retry @ loadable.js?3b89:163
LoadableSubscription @ loadable.js?3b89:130
init @ loadable.js?3b89:54
LoadableImpl @ loadable.js?3b89:80
renderWithHooks @ react-dom.development.js?61bb:14985
updateForwardRef @ react-dom.development.js?61bb:17044
beginWork @ react-dom.development.js?61bb:19098
beginWork$1 @ react-dom.development.js?61bb:23940
performUnitOfWork @ react-dom.development.js?61bb:22776
workLoopSync @ react-dom.development.js?61bb:22707
renderRootSync @ react-dom.development.js?61bb:22670
performSyncWorkOnRoot @ react-dom.development.js?61bb:22293
eval @ react-dom.development.js?61bb:11327
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
flushSyncCallbackQueueImpl @ react-dom.development.js?61bb:11322
flushSyncCallbackQueue @ react-dom.development.js?61bb:11309
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21893
dispatchAction @ react-dom.development.js?61bb:16139
checkForUpdates @ use-subscription.development.js?34f6:85
eval @ loadable.js?3b89:183
_update @ loadable.js?3b89:183
eval @ loadable.js?3b89:164
Promise.then (async)
retry @ loadable.js?3b89:163
LoadableSubscription @ loadable.js?3b89:130
init @ loadable.js?3b89:54
LoadableImpl @ loadable.js?3b89:80
renderWithHooks @ react-dom.development.js?61bb:14985
updateForwardRef @ react-dom.development.js?61bb:17044
beginWork @ react-dom.development.js?61bb:19098
beginWork$1 @ react-dom.development.js?61bb:23940
performUnitOfWork @ react-dom.development.js?61bb:22776
workLoopSync @ react-dom.development.js?61bb:22707
renderRootSync @ react-dom.development.js?61bb:22670
performSyncWorkOnRoot @ react-dom.development.js?61bb:22293
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21881
updateContainer @ react-dom.development.js?61bb:25482
eval @ react-dom.development.js?61bb:26021
unbatchedUpdates @ react-dom.development.js?61bb:22431
legacyRenderSubtreeIntoContainer @ react-dom.development.js?61bb:26020
hydrate @ react-dom.development.js?61bb:26086
renderReactElement @ index.js?20a9:480
doRender @ index.js?20a9:665
_callee2$ @ index.js?20a9:375
tryCatch @ runtime.js?96cf:63
invoke @ runtime.js?96cf:293
eval @ runtime.js?96cf:118
asyncGeneratorStep @ index.js?20a9:28
_next @ index.js?20a9:46
eval @ index.js?20a9:51
eval @ index.js?20a9:43
_render @ index.js?20a9:393
render @ index.js?20a9:396
eval @ next-dev.js?3346:79
eval @ fouc.js?3f0c:12
requestAnimationFrame (async)
displayContent @ fouc.js?3f0c:7
eval @ next-dev.js?3346:78
Promise.then (async)
eval @ next-dev.js?3346:40
./node_modules/next/dist/client/next-dev.js @ main.js?ts=1657796630744:589
options.factory @ webpack.js?ts=1657796630744:697
__webpack_require__ @ webpack.js?ts=1657796630744:37
__webpack_exec__ @ main.js?ts=1657796630744:1399
(anonymous) @ main.js?ts=1657796630744:1400
webpackJsonpCallback @ webpack.js?ts=1657796630744:1280
(anonymous) @ main.js?ts=1657796630744:9
Show 32 more frames
react-dom.development.js?61bb:630 Error: <g> attribute transform: Expected number, "translate(NaN,NaN) rotate(…".
setValueForProperty @ react-dom.development.js?61bb:630
updateDOMProperties @ react-dom.development.js?61bb:8949
updateProperties @ react-dom.development.js?61bb:9368
commitUpdate @ react-dom.development.js?61bb:10252
commitWork @ react-dom.development.js?61bb:21401
commitMutationEffects @ react-dom.development.js?61bb:23401
callCallback @ react-dom.development.js?61bb:3945
invokeGuardedCallbackDev @ react-dom.development.js?61bb:3994
invokeGuardedCallback @ react-dom.development.js?61bb:4056
commitRootImpl @ react-dom.development.js?61bb:23121
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
commitRoot @ react-dom.development.js?61bb:22990
performSyncWorkOnRoot @ react-dom.development.js?61bb:22329
eval @ react-dom.development.js?61bb:11327
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
flushSyncCallbackQueueImpl @ react-dom.development.js?61bb:11322
flushSyncCallbackQueue @ react-dom.development.js?61bb:11309
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21893
dispatchAction @ react-dom.development.js?61bb:16139
checkForUpdates @ use-subscription.development.js?34f6:85
eval @ loadable.js?3b89:183
_update @ loadable.js?3b89:183
eval @ loadable.js?3b89:164
Promise.then (async)
retry @ loadable.js?3b89:163
LoadableSubscription @ loadable.js?3b89:130
init @ loadable.js?3b89:54
LoadableImpl @ loadable.js?3b89:80
renderWithHooks @ react-dom.development.js?61bb:14985
updateForwardRef @ react-dom.development.js?61bb:17044
beginWork @ react-dom.development.js?61bb:19098
beginWork$1 @ react-dom.development.js?61bb:23940
performUnitOfWork @ react-dom.development.js?61bb:22776
workLoopSync @ react-dom.development.js?61bb:22707
renderRootSync @ react-dom.development.js?61bb:22670
performSyncWorkOnRoot @ react-dom.development.js?61bb:22293
eval @ react-dom.development.js?61bb:11327
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
flushSyncCallbackQueueImpl @ react-dom.development.js?61bb:11322
flushSyncCallbackQueue @ react-dom.development.js?61bb:11309
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21893
dispatchAction @ react-dom.development.js?61bb:16139
checkForUpdates @ use-subscription.development.js?34f6:85
eval @ loadable.js?3b89:183
_update @ loadable.js?3b89:183
eval @ loadable.js?3b89:164
Promise.then (async)
retry @ loadable.js?3b89:163
LoadableSubscription @ loadable.js?3b89:130
init @ loadable.js?3b89:54
LoadableImpl @ loadable.js?3b89:80
renderWithHooks @ react-dom.development.js?61bb:14985
updateForwardRef @ react-dom.development.js?61bb:17044
beginWork @ react-dom.development.js?61bb:19098
beginWork$1 @ react-dom.development.js?61bb:23940
performUnitOfWork @ react-dom.development.js?61bb:22776
workLoopSync @ react-dom.development.js?61bb:22707
renderRootSync @ react-dom.development.js?61bb:22670
performSyncWorkOnRoot @ react-dom.development.js?61bb:22293
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21881
updateContainer @ react-dom.development.js?61bb:25482
eval @ react-dom.development.js?61bb:26021
unbatchedUpdates @ react-dom.development.js?61bb:22431
legacyRenderSubtreeIntoContainer @ react-dom.development.js?61bb:26020
hydrate @ react-dom.development.js?61bb:26086
renderReactElement @ index.js?20a9:480
doRender @ index.js?20a9:665
_callee2$ @ index.js?20a9:375
tryCatch @ runtime.js?96cf:63
invoke @ runtime.js?96cf:293
eval @ runtime.js?96cf:118
asyncGeneratorStep @ index.js?20a9:28
_next @ index.js?20a9:46
eval @ index.js?20a9:51
eval @ index.js?20a9:43
_render @ index.js?20a9:393
render @ index.js?20a9:396
eval @ next-dev.js?3346:79
eval @ fouc.js?3f0c:12
requestAnimationFrame (async)
displayContent @ fouc.js?3f0c:7
eval @ next-dev.js?3346:78
Promise.then (async)
eval @ next-dev.js?3346:40
./node_modules/next/dist/client/next-dev.js @ main.js?ts=1657796630744:589
options.factory @ webpack.js?ts=1657796630744:697
__webpack_require__ @ webpack.js?ts=1657796630744:37
__webpack_exec__ @ main.js?ts=1657796630744:1399
(anonymous) @ main.js?ts=1657796630744:1400
webpackJsonpCallback @ webpack.js?ts=1657796630744:1280
(anonymous) @ main.js?ts=1657796630744:9
Show 32 more frames
react-dom.development.js?61bb:630 Error: <path> attribute d: Expected number, "… 16 16 C 16 16, NaN NaN, NaN NaN".
setValueForProperty @ react-dom.development.js?61bb:630
updateDOMProperties @ react-dom.development.js?61bb:8949
updateProperties @ react-dom.development.js?61bb:9368
commitUpdate @ react-dom.development.js?61bb:10252
commitWork @ react-dom.development.js?61bb:21401
commitMutationEffects @ react-dom.development.js?61bb:23401
callCallback @ react-dom.development.js?61bb:3945
invokeGuardedCallbackDev @ react-dom.development.js?61bb:3994
invokeGuardedCallback @ react-dom.development.js?61bb:4056
commitRootImpl @ react-dom.development.js?61bb:23121
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
commitRoot @ react-dom.development.js?61bb:22990
performSyncWorkOnRoot @ react-dom.development.js?61bb:22329
eval @ react-dom.development.js?61bb:11327
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
flushSyncCallbackQueueImpl @ react-dom.development.js?61bb:11322
flushSyncCallbackQueue @ react-dom.development.js?61bb:11309
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21893
dispatchAction @ react-dom.development.js?61bb:16139
checkForUpdates @ use-subscription.development.js?34f6:85
eval @ loadable.js?3b89:183
_update @ loadable.js?3b89:183
eval @ loadable.js?3b89:164
Promise.then (async)
retry @ loadable.js?3b89:163
LoadableSubscription @ loadable.js?3b89:130
init @ loadable.js?3b89:54
LoadableImpl @ loadable.js?3b89:80
renderWithHooks @ react-dom.development.js?61bb:14985
updateForwardRef @ react-dom.development.js?61bb:17044
beginWork @ react-dom.development.js?61bb:19098
beginWork$1 @ react-dom.development.js?61bb:23940
performUnitOfWork @ react-dom.development.js?61bb:22776
workLoopSync @ react-dom.development.js?61bb:22707
renderRootSync @ react-dom.development.js?61bb:22670
performSyncWorkOnRoot @ react-dom.development.js?61bb:22293
eval @ react-dom.development.js?61bb:11327
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
flushSyncCallbackQueueImpl @ react-dom.development.js?61bb:11322
flushSyncCallbackQueue @ react-dom.development.js?61bb:11309
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21893
dispatchAction @ react-dom.development.js?61bb:16139
checkForUpdates @ use-subscription.development.js?34f6:85
eval @ loadable.js?3b89:183
_update @ loadable.js?3b89:183
eval @ loadable.js?3b89:164
Promise.then (async)
retry @ loadable.js?3b89:163
LoadableSubscription @ loadable.js?3b89:130
init @ loadable.js?3b89:54
LoadableImpl @ loadable.js?3b89:80
renderWithHooks @ react-dom.development.js?61bb:14985
updateForwardRef @ react-dom.development.js?61bb:17044
beginWork @ react-dom.development.js?61bb:19098
beginWork$1 @ react-dom.development.js?61bb:23940
performUnitOfWork @ react-dom.development.js?61bb:22776
workLoopSync @ react-dom.development.js?61bb:22707
renderRootSync @ react-dom.development.js?61bb:22670
performSyncWorkOnRoot @ react-dom.development.js?61bb:22293
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21881
updateContainer @ react-dom.development.js?61bb:25482
eval @ react-dom.development.js?61bb:26021
unbatchedUpdates @ react-dom.development.js?61bb:22431
legacyRenderSubtreeIntoContainer @ react-dom.development.js?61bb:26020
hydrate @ react-dom.development.js?61bb:26086
renderReactElement @ index.js?20a9:480
doRender @ index.js?20a9:665
_callee2$ @ index.js?20a9:375
tryCatch @ runtime.js?96cf:63
invoke @ runtime.js?96cf:293
eval @ runtime.js?96cf:118
asyncGeneratorStep @ index.js?20a9:28
_next @ index.js?20a9:46
eval @ index.js?20a9:51
eval @ index.js?20a9:43
_render @ index.js?20a9:393
render @ index.js?20a9:396
eval @ next-dev.js?3346:79
eval @ fouc.js?3f0c:12
requestAnimationFrame (async)
displayContent @ fouc.js?3f0c:7
eval @ next-dev.js?3346:78
Promise.then (async)
eval @ next-dev.js?3346:40
./node_modules/next/dist/client/next-dev.js @ main.js?ts=1657796630744:589
options.factory @ webpack.js?ts=1657796630744:697
__webpack_require__ @ webpack.js?ts=1657796630744:37
__webpack_exec__ @ main.js?ts=1657796630744:1399
(anonymous) @ main.js?ts=1657796630744:1400
webpackJsonpCallback @ webpack.js?ts=1657796630744:1280
(anonymous) @ main.js?ts=1657796630744:9
Show 32 more frames
react-dom.development.js?61bb:630 Error: <g> attribute transform: Expected number, "translate(NaN,NaN) rotate(…".
setValueForProperty @ react-dom.development.js?61bb:630
updateDOMProperties @ react-dom.development.js?61bb:8949
updateProperties @ react-dom.development.js?61bb:9368
commitUpdate @ react-dom.development.js?61bb:10252
commitWork @ react-dom.development.js?61bb:21401
commitMutationEffects @ react-dom.development.js?61bb:23401
callCallback @ react-dom.development.js?61bb:3945
invokeGuardedCallbackDev @ react-dom.development.js?61bb:3994
invokeGuardedCallback @ react-dom.development.js?61bb:4056
commitRootImpl @ react-dom.development.js?61bb:23121
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
commitRoot @ react-dom.development.js?61bb:22990
performSyncWorkOnRoot @ react-dom.development.js?61bb:22329
eval @ react-dom.development.js?61bb:11327
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
flushSyncCallbackQueueImpl @ react-dom.development.js?61bb:11322
flushSyncCallbackQueue @ react-dom.development.js?61bb:11309
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21893
dispatchAction @ react-dom.development.js?61bb:16139
checkForUpdates @ use-subscription.development.js?34f6:85
eval @ loadable.js?3b89:183
_update @ loadable.js?3b89:183
eval @ loadable.js?3b89:164
Promise.then (async)
retry @ loadable.js?3b89:163
LoadableSubscription @ loadable.js?3b89:130
init @ loadable.js?3b89:54
LoadableImpl @ loadable.js?3b89:80
renderWithHooks @ react-dom.development.js?61bb:14985
updateForwardRef @ react-dom.development.js?61bb:17044
beginWork @ react-dom.development.js?61bb:19098
beginWork$1 @ react-dom.development.js?61bb:23940
performUnitOfWork @ react-dom.development.js?61bb:22776
workLoopSync @ react-dom.development.js?61bb:22707
renderRootSync @ react-dom.development.js?61bb:22670
performSyncWorkOnRoot @ react-dom.development.js?61bb:22293
eval @ react-dom.development.js?61bb:11327
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
flushSyncCallbackQueueImpl @ react-dom.development.js?61bb:11322
flushSyncCallbackQueue @ react-dom.development.js?61bb:11309
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21893
dispatchAction @ react-dom.development.js?61bb:16139
checkForUpdates @ use-subscription.development.js?34f6:85
eval @ loadable.js?3b89:183
_update @ loadable.js?3b89:183
eval @ loadable.js?3b89:164
Promise.then (async)
retry @ loadable.js?3b89:163
LoadableSubscription @ loadable.js?3b89:130
init @ loadable.js?3b89:54
LoadableImpl @ loadable.js?3b89:80
renderWithHooks @ react-dom.development.js?61bb:14985
updateForwardRef @ react-dom.development.js?61bb:17044
beginWork @ react-dom.development.js?61bb:19098
beginWork$1 @ react-dom.development.js?61bb:23940
performUnitOfWork @ react-dom.development.js?61bb:22776
workLoopSync @ react-dom.development.js?61bb:22707
renderRootSync @ react-dom.development.js?61bb:22670
performSyncWorkOnRoot @ react-dom.development.js?61bb:22293
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21881
updateContainer @ react-dom.development.js?61bb:25482
eval @ react-dom.development.js?61bb:26021
unbatchedUpdates @ react-dom.development.js?61bb:22431
legacyRenderSubtreeIntoContainer @ react-dom.development.js?61bb:26020
hydrate @ react-dom.development.js?61bb:26086
renderReactElement @ index.js?20a9:480
doRender @ index.js?20a9:665
_callee2$ @ index.js?20a9:375
tryCatch @ runtime.js?96cf:63
invoke @ runtime.js?96cf:293
eval @ runtime.js?96cf:118
asyncGeneratorStep @ index.js?20a9:28
_next @ index.js?20a9:46
eval @ index.js?20a9:51
eval @ index.js?20a9:43
_render @ index.js?20a9:393
render @ index.js?20a9:396
eval @ next-dev.js?3346:79
eval @ fouc.js?3f0c:12
requestAnimationFrame (async)
displayContent @ fouc.js?3f0c:7
eval @ next-dev.js?3346:78
Promise.then (async)
eval @ next-dev.js?3346:40
./node_modules/next/dist/client/next-dev.js @ main.js?ts=1657796630744:589
options.factory @ webpack.js?ts=1657796630744:697
__webpack_require__ @ webpack.js?ts=1657796630744:37
__webpack_exec__ @ main.js?ts=1657796630744:1399
(anonymous) @ main.js?ts=1657796630744:1400
webpackJsonpCallback @ webpack.js?ts=1657796630744:1280
(anonymous) @ main.js?ts=1657796630744:9
Show 32 more frames
react-dom.development.js?61bb:630 Error: <path> attribute d: Expected number, "… 16 16 C 16 16, NaN NaN, NaN NaN".
setValueForProperty @ react-dom.development.js?61bb:630
updateDOMProperties @ react-dom.development.js?61bb:8949
updateProperties @ react-dom.development.js?61bb:9368
commitUpdate @ react-dom.development.js?61bb:10252
commitWork @ react-dom.development.js?61bb:21401
commitMutationEffects @ react-dom.development.js?61bb:23401
callCallback @ react-dom.development.js?61bb:3945
invokeGuardedCallbackDev @ react-dom.development.js?61bb:3994
invokeGuardedCallback @ react-dom.development.js?61bb:4056
commitRootImpl @ react-dom.development.js?61bb:23121
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
commitRoot @ react-dom.development.js?61bb:22990
performSyncWorkOnRoot @ react-dom.development.js?61bb:22329
eval @ react-dom.development.js?61bb:11327
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
flushSyncCallbackQueueImpl @ react-dom.development.js?61bb:11322
flushSyncCallbackQueue @ react-dom.development.js?61bb:11309
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21893
dispatchAction @ react-dom.development.js?61bb:16139
checkForUpdates @ use-subscription.development.js?34f6:85
eval @ loadable.js?3b89:183
_update @ loadable.js?3b89:183
eval @ loadable.js?3b89:164
Promise.then (async)
retry @ loadable.js?3b89:163
LoadableSubscription @ loadable.js?3b89:130
init @ loadable.js?3b89:54
LoadableImpl @ loadable.js?3b89:80
renderWithHooks @ react-dom.development.js?61bb:14985
updateForwardRef @ react-dom.development.js?61bb:17044
beginWork @ react-dom.development.js?61bb:19098
beginWork$1 @ react-dom.development.js?61bb:23940
performUnitOfWork @ react-dom.development.js?61bb:22776
workLoopSync @ react-dom.development.js?61bb:22707
renderRootSync @ react-dom.development.js?61bb:22670
performSyncWorkOnRoot @ react-dom.development.js?61bb:22293
eval @ react-dom.development.js?61bb:11327
unstable_runWithPriority @ scheduler.development.js?3069:468
runWithPriority$1 @ react-dom.development.js?61bb:11276
flushSyncCallbackQueueImpl @ react-dom.development.js?61bb:11322
flushSyncCallbackQueue @ react-dom.development.js?61bb:11309
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21893
dispatchAction @ react-dom.development.js?61bb:16139
checkForUpdates @ use-subscription.development.js?34f6:85
eval @ loadable.js?3b89:183
_update @ loadable.js?3b89:183
eval @ loadable.js?3b89:164
Promise.then (async)
retry @ loadable.js?3b89:163
LoadableSubscription @ loadable.js?3b89:130
init @ loadable.js?3b89:54
LoadableImpl @ loadable.js?3b89:80
renderWithHooks @ react-dom.development.js?61bb:14985
updateForwardRef @ react-dom.development.js?61bb:17044
beginWork @ react-dom.development.js?61bb:19098
beginWork$1 @ react-dom.development.js?61bb:23940
performUnitOfWork @ react-dom.development.js?61bb:22776
workLoopSync @ react-dom.development.js?61bb:22707
renderRootSync @ react-dom.development.js?61bb:22670
performSyncWorkOnRoot @ react-dom.development.js?61bb:22293
scheduleUpdateOnFiber @ react-dom.development.js?61bb:21881
updateContainer @ react-dom.development.js?61bb:25482
eval @ react-dom.development.js?61bb:26021
unbatchedUpdates @ react-dom.development.js?61bb:22431
legacyRenderSubtreeIntoContainer @ react-dom.development.js?61bb:26020
hydrate @ react-dom.development.js?61bb:26086
renderReactElement @ index.js?20a9:480
doRender @ index.js?20a9:665
_callee2$ @ index.js?20a9:375
tryCatch @ runtime.js?96cf:63
invoke @ runtime.js?96cf:293
eval @ runtime.js?96cf:118
asyncGeneratorStep @ index.js?20a9:28
_next @ index.js?20a9:46
eval @ index.js?20a9:51
eval @ index.js?20a9:43
_render @ index.js?20a9:393
render @ index.js?20a9:396
eval @ next-dev.js?3346:79
eval @ fouc.js?3f0c:12
requestAnimationFrame (async)
displayContent @ fouc.js?3f0c:7
eval @ next-dev.js?3346:78
Promise.then (async)
eval @ next-dev.js?3346:40
./node_modules/next/dist/client/next-dev.js @ main.js?ts=1657796630744:589
options.factory @ webpack.js?ts=1657796630744:697
__webpack_require__ @ webpack.js?ts=1657796630744:37
__webpack_exec__ @ main.js?ts=1657796630744:1399
(anonymous) @ main.js?ts=1657796630744:1400
webpackJsonpCallback @ webpack.js?ts=1657796630744:1280
(anonymous) @ main.js?ts=1657796630744:9
Show 32 more frames
react-dom.development.js?61bb:630 Error: <g> attribute transform: Expected number, "translate(NaN,NaN) rotate(…".

**Comment by peersky** [comment]
image
Code block overflows on smaller screen

**Comment by peersky** [comment]
Cards on mobile screen have different height (screenshot is done without mouse hovering)
image

**Review by peersky — CHANGES_REQUESTED** [review]
Cleanup of comments and small improvements are desired

File: frontend/src/core/providers/OverlayProvider/index.js (hunk 5 above)
Comment (kellan-simiotics) [reply frontend/src/core/providers/OverlayProvider/index.js]: removed

File: frontend/src/core/providers/OverlayProvider/index.js (hunk 4 above)
Comment (kellan-simiotics) [reply frontend/src/core/providers/OverlayProvider/index.js]: removed

File: frontend/pages/contact/index.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+      />
+    </Box>
+  </Box>
+);
+
+Contact.getLayout = getLayout;
+
+export async function getStaticProps() {
+  const layoutProps = getLayoutProps();
+  // const metaTags = {
```
[the remark this answers is not in the document]
Comment (kellan-simiotics) [reply frontend/pages/contact/index.js:41]: > Implement contact form metatags and remove comments

Done.

**Comment by kellan-simiotics** [comment]
> Cards on mobile screen have different height (screenshot is done without mouse hovering) image

Fixed.

**Review by peersky — APPROVED** [review]
gg

**Pull request merged**